<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPayeesView</class>
 <widget class="QWidget" name="KPayeesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>567</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="groupBox1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Your payees</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="m_newButton">
           <property name="text">
            <string>New</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_renameButton">
           <property name="text">
            <string>Ren</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_deleteButton">
           <property name="text">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_mergeButton">
           <property name="text">
            <string>Merge</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="m_listTopHLayout">
         <item>
          <widget class="QLineEdit" name="m_searchWidget"/>
         </item>
         <item>
          <widget class="KComboBox" name="m_filterBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="m_payees">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="m_tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="transactionTab">
       <attribute name="title">
        <string>Transactions</string>
       </attribute>
       <layout class="QVBoxLayout" stretch="0,0">
        <item>
         <widget class="LedgerView" name="m_register"/>
        </item>
        <item>
         <widget class="QLabel" name="m_balanceLabel">
          <property name="text">
           <string>Balance: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="addressTab">
       <attribute name="title">
        <string comment="Street address">Address</string>
       </attribute>
       <layout class="QGridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="5" column="0">
         <widget class="QLabel" name="TextLabel7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string comment="E-mail address">E-Mail:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="TextLabel5_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>City:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TextLabel5_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>State:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="TextLabel5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Postal Code:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="TextLabel6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Telephone/Fax:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="TextLabel4_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Notes</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="KLineEdit" name="emailEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="TextLabel4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string comment="Street address">Address:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="m_sendMail">
          <property name="toolTip">
           <string>Creates new e-mail to your payee.</string>
          </property>
          <property name="whatsThis">
           <string>Use this to create new e-mail to your payee.</string>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="KLineEdit" name="telephoneEdit"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="KLineEdit" name="payeecityEdit"/>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="KLineEdit" name="payeestateEdit"/>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="KLineEdit" name="postcodeEdit"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="KTextEdit" name="addressEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="KTextEdit" name="notesEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="spacer6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="matchingTab">
       <attribute name="title">
        <string>Matching</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Match method</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="matchTypeCombo"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkMatchIgnoreCase">
            <property name="text">
             <string>Ignore Case</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>List of matching names</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="KEditListWidget" name="matchKeyEditList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="whatsThis">
              <string>This list contains the names that will match this payee if a transaction is imported from an external source. Keep in mind, that you can specify regular expressions here.</string>
             </property>
             <property name="title" stdset="0">
              <string>List of matching names</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="defaultAccountTab">
       <attribute name="title">
        <string>Default Category</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="checkEnableDefaultCategory">
          <property name="text">
           <string>Use the default category for new transactions with this payee</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDefaultCategory">
            <property name="text">
             <string>Default category:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KMyMoneyAccountCombo" name="comboDefaultCategory"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonSuggestACategory">
            <property name="text">
             <string>Suggest a category</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPayeeIdentifiers">
       <attribute name="title">
        <string>Account Numbers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="KPayeeIdentifierView" name="payeeIdentifiers" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_helpButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>260</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_syncAddressbook">
       <property name="toolTip">
        <string>Fetches the payee's data from your addressbook.</string>
       </property>
       <property name="whatsThis">
        <string>Use this to fetch payee's data.</string>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_updateButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Accepts the entered data and stores it</string>
       </property>
       <property name="whatsThis">
        <string>Use this to accept the modified data.</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyAccountCombo</class>
   <extends>KComboBox</extends>
   <header>kmymoneyaccountcombo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPayeeIdentifierView</class>
   <extends>QWidget</extends>
   <header>kpayeeidentifierview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LedgerView</class>
   <extends>QTableView</extends>
   <header>ledgerview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabWidget</tabstop>
  <tabstop>m_filterBox</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>payeecityEdit</tabstop>
  <tabstop>payeestateEdit</tabstop>
  <tabstop>postcodeEdit</tabstop>
  <tabstop>telephoneEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>m_sendMail</tabstop>
  <tabstop>notesEdit</tabstop>
  <tabstop>matchTypeCombo</tabstop>
  <tabstop>checkMatchIgnoreCase</tabstop>
  <tabstop>checkEnableDefaultCategory</tabstop>
  <tabstop>comboDefaultCategory</tabstop>
  <tabstop>buttonSuggestACategory</tabstop>
  <tabstop>m_helpButton</tabstop>
  <tabstop>m_syncAddressbook</tabstop>
  <tabstop>m_updateButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">../widgets/kmymoneyaccountcombo.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
